apps/acceptance/clock/acceptanceStopwatch.js

```diff
@@ -0,0 +1,71 @@
+'use strict';
+var Clock = require('./regions/appView');
+var expect = require('chai').expect;
+
+marionette('Clock Acceptance Suite', function(){
+    var client;
+    var clock;
+    client = marionette.client({
+        prefs: {
+            // we need to disable the keyboard
+            // to avoid intermittent failures on
+            // Travis (transitions might take
+            // longer to run and block UI)
+            'dom.mozInputMethod.enabled': false,
+            // Do not require the B2G-desktop app
+            // window to have focus (as per the
+            // system window manager) in order for
+            // it to do focus-related things.
+            'focusmanager.testmode': true
+        },
+        settings: {
+            'ftu.manifestURL': null,
+            'lockscreen.enabled': false
+        }
+    });
+    setup(function() {
+        clock = new Clock(client);
+        clock.launch();
+    });
+    suite('Stopwatch Suite', function() {
+        test('Start Stopwatch and Reset', function() {
+            var stopwatchView = clock.switchView('stopwatch');
```

Comment:
Here too, we can be more closer to a user's intent: `stopwatch = clock.goToStopwatch()`. Defining "view" in the test sounds like a implementation detail, and will be redundant. Nearly every test will manipulate views. Precising it every time will introduce fatigue to the reader and he will likely start ignoring some parts of variable names, all along the tests. Finding bugs in there will become more difficult.